Frameworks/Foundation/NSUserDefaults.mm

```diff
@@ -48,6 +48,7 @@
 
 @implementation NSUserDefaults {
     StrongId<NSMutableDictionary> _registrationDict;
+    StrongId<NSOperationQueue> _synchronizeQueue;
 }
```

Comment:
the proper implementation of NSOperationQueue would use thread pools, but agree that our current implementation is not efficient.  

---
In reply to: [69786383](https://github.com/Microsoft/WinObjC/pull/612#discussion_r69786383) [](ancestors = 69786383)